### DISCUSSION TITLE
installation guide for binaries?

### DISCUSSION THREAD

**Discussion by CarmenTrigel** [gsamokovarov/jump#61] [Q&A]
How to integrate the binaries (e.g. arm) into the system: where to copy the binary, folders to create, etc.

**Comment by gsamokovarov** [comment]
You have to download the linux ARM binary from https://github.com/gsamokovarov/jump/releases/tag/v0.40.0 and put it somewhere in `$PATH` as `jump`, say in `/usr/local/bin/jump`. Something that I do is to have `~/bin` in `$PATH`, so I can have per-user binaries without the need of sudo access to install them in places like `/usr/local/bin`.

I don't have a build of the M1 Macs at the moment, though. I'm not really sure if `brew` has this covered at the moment. If you need an ARM mac build, respond to this thread and I'll build it for you.

**Reply by CarmenTrigel** [reply]
I use "jump" on a Pi4 (and others...). I create a symbolic link in `/usr/bin` to the binary in `/opt/bin`, where all my scripts an binaries are located and bind jump in `~./bashrc`. I figured it out by myself, but perhaps you could mention it in your instructions.
Thank you for jump - I switched from "autojump" due to too many fat-finger errors ;-)

**Reply by gsamokovarov** [reply]
Thanks, I'll update the readme. I was using autojump before and ended up with my own solution exactly because of my fat fingers. 😅 The fuzzy matching is just so much better!
